# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: parser/html/nsHtml5TreeOpStage.h
# Commit: 77388b0795b4
# Full Hash: 77388b0795b4f9a30171c922d6b649c52e5f6c87
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-23 09:34:36
# Description:
#   Bug 1745142 - Communicate encoding commitment via speculative load queue. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133996
# ==============================================================================

diff -r a388127095db -r 77388b0795b4 parser/html/nsHtml5TreeOpStage.h
--- a/parser/html/nsHtml5TreeOpStage.h	Wed Dec 22 15:47:08 2021 +0000
+++ b/parser/html/nsHtml5TreeOpStage.h	Wed Dec 22 15:54:49 2021 +0000
@@ -24,6 +24,11 @@
   virtual void MoveOpsFrom(nsTArray<nsHtml5TreeOperation>& aOpQueue) override;
 
   /**
+   * Retrieve the staged operations into the argument.
+   */
+  void MoveOpsTo(nsTArray<nsHtml5TreeOperation>& aOpQueue);
+
+  /**
    * Retrieve the staged operations and speculative loads into the arguments.
    */
   void MoveOpsAndSpeculativeLoadsTo(